sites/kit.svelte.dev/scripts/types/index.js

```diff
@@ -1,9 +1,10 @@
-import fs from 'fs';
-import path from 'path';
+// @ts-check
```

Comment:
could we configure the tsconfig to check this file instead of having to annotate it?

Reply:
Done!